server/src/main/java/org/eclipse/openvsx/entities/ExtensionVersion.java

```diff
@@ -26,6 +26,7 @@
 import org.eclipse.openvsx.util.TimeUtil;
 
 @Entity
+@Table(uniqueConstraints = { @UniqueConstraint(columnNames = { "extension_id", "version" })})
```

Comment:
`ExtensionVersion` object does not seem to have `extension_id` property.

Is it a list of SQL fields here?

Reply:
It's a mix of both. When the property isn't another entity you can use the property name (`version`, `publicId`). But, when the property does refer to another entity, then you have to use the SQL mapping column (property `extension` in `ExtensionVersion` is mapped by `extension_id` in the `extension_version` table).